app/helpers/application_helper.rb

```diff
@@ -2,9 +2,9 @@ module ApplicationHelper
   def navigation_items
     items = []
 
-    items << { text: "Govspeak Preview", href: "/", active: is_current?("/") }
-    items << { text: "Govspeak Guide", href: "/guide", active: is_current?("/guide") }
-    items << { text: "Convert Google Doc to Govspeak", href: "/convert", active: is_current?("/convert") }
+    items << { text: "Govspeak converter", href: "/", active: is_current?("/") }
+    items << { text: "Govspeak guide", href: "/guide", active: is_current?("/guide") }
```

Comment:
Is it easy to update the title as the user navigates? I feel like the title should shift between:

```
Govspeak preview
Govspeak preview - Govspeak guide
Govspeak preview - Convert Google Docs to Govspeak and preview
```

Reply:
Good point. The only issue is that the home page is essentially the /preview page and so uses the same page title. I've now updated the title on each page to match the `<h1>`.